pina/solvers/pinn.py

```diff
@@ -95,8 +97,14 @@ def configure_optimizers(self):
         :return: The optimizers and the schedulers
         :rtype: tuple(list, list)
         """
+        # if the problem is an InverseProblem, add the unknown parameters
+        # to the parameters that the optimizer needs to optimize
+        if isinstance(self.problem, InverseProblem):
```

Comment:
What about if you used a different solver? This goes in SolverInterface configure_optimizer, then we call the super.